app/controllers/RouterController.scala

```diff
@@ -99,7 +99,23 @@ class RouterController @Inject()(val authConnector: AuthConnector,
 
         def hasPTEnrolment: Boolean = enrolments.nonEmpty && enrolments.map(_.key).contains("HMRC-PT")
 
-        if(hasPTEnrolment){
+        if (isAffinity(Organisation)) {
+          if(hasPTEnrolment){
+            route("user-having-OrganisationAffinity-and-PTEnrolment-would-have-gone-to-PTA", "BTA", user)
+          } else if (isNotGateway) {
+            route("user-having-OrganisationAffinity-and-is-not-a-gateway-user-would-have-gone-to-PTA", "BTA", user)
+          } else if (isAdmin) {
+              if (hasOnlySAenrolments) {
+                if (isVerified)
+                  route("user-with-OrganisationAffinity-Admin-sa-enrolments-and-cl250-would-have-gone-to-PTA", "BTA", user)
+                else
+                  route("user-with-OrganisationAffinity-Admin-sa-enrolments-and-not-cl250-would-have-gone-to-PTA", "BTA", user)
```

Comment:
`user-with-OrganisationAffinity-Admin-sa-enrolments-and-not-cl250-would-have-gone-to-BTA` not PTA